analytics/terraform/emr-eks-karpenter/addons.tf

```diff
@@ -1,10 +1,15 @@
+
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.19.0"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.25.0"
```

Comment:
what was the decision on add-on module? I thought we gonna use a new repo just for add-on?

Reply:
The source will be changed once the add-ons repo is published under aws-ia org. No change to the code. Our add-ons modules do a lot of heavy lifting so lets keep it this way.